oauth2/oauth2-resource-server/src/test/java/org/springframework/security/oauth2/server/resource/web/DefaultBearerTokenResolverTests.java

```diff
@@ -43,6 +43,15 @@ public void setUp() {
 		this.resolver = new DefaultBearerTokenResolver();
 	}
 
+	@Test
+	public void resolveWhenNoTokenIsPresentThenTokenIsNotResolved() {
```

Comment:
There is semantic difference that in new test case we have `setAllowFormEncodedBodyParameter(true)` and `setAllowUriQueryParameter(true)` and still it will not resolve the token.

Reply:
I don't see how there's a semantic difference when you're not actually passing anything with the request to resolve. This test only verifies that the `DefaultBearerTokenResolver#resolve` returns `null` when there's nothing to resolve (same as `resolveWhenNoHeaderIsPresentThenTokenIsNotResolved`) and that is the same regardless of how you configure `DefaultBearerTokenResolver`.

Reply:
Both input and expected output is same but this test case gives coverage to different branch logic when form or query parameter is enabled.

Reply:
If you still think this is not useful, I will remove this test case once you confirm.

Reply:
I will remove test case `resolveWhenNoTokenIsPresentThenTokenIsNotResolved()`